### DISCUSSION TITLE
List of CBC parameters

### DISCUSSION THREAD

**Discussion by anselmo-pitombeira** [coin-or/Cbc#565] [Q&A]
I am working with CBC via Python Pyomo. I want to set the parameter in CBC which specifies as stopping criterion the maximum seconds without improving the feasible solution, but I do not know how to do this. Haroldo Gambini has collected some options in a document entitlet "A Short Guide to the CBC Command Line Interface", but this is from 2017 and does not include the abovementioned parameter. (I have seen that Gambini has implemented this parameter in the source code.) I am using Windows binaries from the 2.10.8 release.

**Comment by guifcoelho** [comment]
Hi! Take a look at [this post](https://github.com/coin-or/Cbc/discussions/379). Call Cbc and enter '?'. It will return all available options.
